tests/test_amqp_reporter.py

```diff
@@ -15,6 +15,14 @@
 
 import pytest
 
+from ensembl.production.reporting.config import config
+from ensembl.production.reporting.amqp_reporter import validate_payload, compose_email
+
+
+SMTP_HOST = "127.0.0.1"
```

Comment:
Get a config var instead?

Reply:
This global variable is specific to the test suite only as it is used by `SMTPController()` to start the SMTP test server. Since tests have only a couple of variables I didn't think it would be necessary to have a separated config file for it, and I don't want to add test specific config in the reporter config module. I'll rename the variable to make it more clear though.

Reply:
Sorry I mixed up with previous amqp_reporter.py file. So That's ok.